src/engine/Core.java

```diff
@@ -175,7 +175,8 @@ public static void main(final String[] args) {
 							gameState.getTime(),
 							gameState.getCoin() + roundState.getRoundCoin(),
 							gameState.getGem());
```

Comment:
GameState 객체 생성하는 부분에 구문 오류가 있네요. 
```suggestion
							gameState.getGem(),
```